portal-api/src/v1/validation/fields/issued-date.js

```diff
@@ -22,45 +19,43 @@ module.exports = (facility, errorList, deal) => {
     manualInclusionNoticeSubmissionDate: minSubmissionDateTimestamp,
   } = deal.details;
 
-  const dealSubmissionDate = formattedTimestamp(dealSubmissionDateTimestamp);
-  const minSubmissionDate = formattedTimestamp(minSubmissionDateTimestamp);
-  const issuedDate = formattedTimestamp(facility.issuedDate);
-  const today = moment();
+  const dealSubmissionDate = new Date(Number.parseInt(dealSubmissionDateTimestamp, 10));
+  const minSubmissionDate = new Date(Number.parseInt(minSubmissionDateTimestamp, 10));
+  const issuedDate = new Date(Number.parseInt(facility.issuedDate, 10));
+  const today = new Date();
 
-  let dateOfSubmission = moment().utc().valueOf();
+  let dateOfSubmission = today.valueOf();
 
   if (dealSubmissionDateTimestamp || minSubmissionDateTimestamp) {
     if (submissionType === CONSTANTS.DEAL.SUBMISSION_TYPE.AIN) {
-      dateOfSubmission = dealSubmissionDate;
+      dateOfSubmission = new Date(dealSubmissionDate.valueOf());
     } else if (minSubmissionDateTimestamp) {
-      dateOfSubmission = minSubmissionDate;
+      dateOfSubmission = new Date(minSubmissionDate.valueOf);
```

Comment:
```suggestion
      dateOfSubmission = new Date(minSubmissionDate.valueOf());
```

concerning that the tests pass with this

Reply:
This was to clone the date to avoid any confusing reference shenanigans. I think I can just bypass declaring `minSubmissionDate` at all though